pyproject.toml

```diff
@@ -15,14 +15,12 @@ keywords = ["vector", "search", "neural", "matching", "client"]
 
 [tool.poetry.dependencies]
 python = ">=3.7,<4.0"
-httpx = { version = "^0.23.0", extras = ["http2"] }
-numpy = "^1.21"
+httpx = { version = ">=0.14.0", extras = ["http2"] }
```

Comment:
can it be at least `0.13.3`?

Reply:
if this is critical for your use-case, you can always just make raw http requests without any client